test/src/blind_auction.vy

```diff
@@ -0,0 +1,177 @@
+# Blind Auction # Adapted to Vyper from [Solidity by Example](https://github.com/ethereum/solidity/blob/develop/docs/solidity-by-example.rst#blind-auction-1)
```

Comment:
This is nice but probably too complex to use as examples, let's use the little vyper contracts we made snek for as the examples instead

Reply:
Attribution is here, not solidity original source. I thought you were the one who adopted it based on your comment. Please be mindful of code authorship attributions. I do not see a license so I don't even know if this is legal technically.
https://vyper.readthedocs.io/en/stable/vyper-by-example.html

Reply:
I didn't add that comment, the files added for integration tests are copied from https://github.com/vyperlang/vyper/tree/master/examples/auctions . The vyper repo has apache licence.

Reply:
so that’s the link that should be in the file
in any case I think we should ship a smaller example contract